students/stuartkershaw/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,31 @@
+def exchange_first_last(seq):
+    seq[0], seq[-1] = seq[-1], seq[0]
+    return seq
```

Comment:
neat solution!

how would you solve using slicing only? (seq[-1], seq[0] is indexing, where slicing is seq[start: end: step])